gradle.properties

```diff
@@ -4,13 +4,13 @@ java_version=17
 
 # Paper
     paper_api_version=1.19
-    paper_version=1.19.2-R0.1-SNAPSHOT
+    paper_version=1.19.4-R0.1-SNAPSHOT
 
 # Fabric
-    minecraft_version=1.19.2
-    yarn_mappings=1.19.2+build.28
+    minecraft_version=1.19.4
+    yarn_mappings=1.19.4+build.2
     loader_version=0.14.19
-    fabric_version=0.76.0+1.19.2
+    fabric_version=0.80.0+1.19.4
```

Comment:
We want to target 1.19.2+, it might be better to target 1.19.2 here to ensure we don't use APIs not available in previous versions, but it's probably fine

Reply:
Will be done in the next commit.